.pre-commit-config.yaml

```diff
@@ -20,6 +20,13 @@ repos:
           - --expand-star-imports
           - --remove-duplicate-keys
           - --remove-unused-variables
+  - repo: https://github.com/Pierre-Sassoulas/copyright_notice_precommit
+    rev: 6f5a59c3e1cb0f20731eb1ff32622c9a2fc72d9a
+    hooks:
+      - id: copyright-notice
+        args: ["--notice=copyright.txt", "--enforce-all", "--autofix"]
+        exclude: tests/testdata
+        types: [python]
```

Comment:
What do you guys think about adding it only to Python files inside `astroid/*`?
IMO there isn't much value in adding it to test files or custom scripts.

Reply:
Fine with me!

Reply:
Discussed [here](https://github.com/PyCQA/astroid/pull/1441#discussion_r820155488)